Jenkinsfile

```diff
@@ -523,7 +523,7 @@ def e2e(Map args = [:]) {
   if (args.e2e.get('entrypoint', '')?.trim()) {
     e2e_with_entrypoint(args)
   } else {
-    runE2E(testMatrixFile: args.e2e?.get('testMatrixFile', ''),
+    runE2E(testMatrixFile: '.ci/.e2e-tests-beats.yaml',
```

Comment:
Because we should simply call the Beats profile, I'd suggest removing the testMatrixFile atribute in the `args.e2e` object, if it existed